test-apps/basic/src/index.tsx

```diff
@@ -14,6 +14,16 @@ let test_function = (word) => {
 }
 
 const App = () => {
+
+  let testObject = {
+    someFunction() {
+      let test_flag = "test_flag_within_function_within_object";
```

Comment:
Can you change this to perform an API call with `asApp()`? That way, we can check if we're also resolving in the interpreter/checkers.

Reply:
Fixed. Thank you!